src/System.Management.Automation/engine/remoting/common/WireDataFormat/EncodeAndDecode.cs

```diff
@@ -1196,10 +1196,10 @@ internal static RemoteDataObject GenerateCreatePowerShell(ClientRemotePowerShell
             // Add Reason property
             if (stateInfo.Reason != null)
             {
-                // If Reason is of not type IContainsErrorRecord, a new ErrorRecord is
-                // created using this errorId
```

Comment:
Why do we remove the comment?

Reply:
The logic is encapsulated in `GetExceptionProperty` so the comment is moved to method remarks there.